src/ic/url.ml

```diff
@@ -79,10 +81,12 @@ let parse (f: string) : (parsed, string) result =
       match Lib.String.chop_prefix "canister:" f with
       | Some suffix -> Ok (IcAlias suffix)
       | None ->
-        begin match Stdlib.String.index_opt f ':' with
-        | Some _ -> Error "Unrecognized URL"
-        | None -> Ok (Relative (Lib.FilePath.normalise f))
-        end
+        match Lib.String.chop_prefix "blob:file:" f with
```

Comment:
Hmm, if this is meant to be a URI, then I don't think you can have a colon within the scheme."blob.file:" would be ok as is "mo.blob:"

Reply:
Oh ok, I was looking in the wrong place perhaps: https://datatracker.ietf.org/doc/html/rfc3986#section-3.1